frontend/__tests__/components/buttons.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { createRemixStub } from '@remix-run/testing';
+
+import { describe, expect, it, vi } from 'vitest';
+
+import { Button, ButtonLink } from '~/components/buttons';
+
+describe('Button Component', () => {
+  it('renders button with default props', () => {
+    const { getByText } = render(<Button>Click me</Button>);
+    const button = getByText('Click me');
```

Comment:
Why aren't you using `data-testid` for this?

Reply:
Since I have control over the button content I don't see the need of using `data-testid` and it assert that `children` is rendered correctly.